fixtures/simple/apt.yml

```diff
@@ -3,6 +3,10 @@ keys:
 - {{.repoBaseURL}}/gpg_public_key
 repos:
 - deb {{.repoBaseURL}} trusty main
+- deb {{.repoBaseURL}} trusty-backports main
+priorities:
```

Comment:
Could we support these objects in the `repos` map instead of creating a separate map for them?

Reply:
My thinking with creating a new mapping was to maintain backward compatibility with existing `apt.yml` files. I also like the fact that each entry in the `repos` map corresponds exactly to a line in the `sources.list` file. Putting the priority specifications in a separate map also made the logic that optionally writes the `preferences` file much cleaner.

That being said, if you'd really like to have everything in the `repos` map, I could certainly do that, but it would be difficult to do that without breaking the current YAML format.

Let me know!